bin/bootstrap.sh

```diff
@@ -15,24 +15,28 @@ set -ev
 ROOT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && cd .. && pwd)"
 cd $ROOT_DIR
 
+printf 'Installing dependencies...\n'
+yarn install --immutable
+printf '\n'
```

Comment:
The same concept as https://github.com/zooniverse/front-end-monorepo/pull/6393. Install once, and then build the rest of the packages. This brings the bootstrap file up to date with the CI build scripts.